### ISSUE TITLE
Graphic glitches with Vox Populi full version

### ISSUE THREAD

**Issue by kjades** [LoneGazebo/Community-Patch-DLL#2973]
1 Latest version of the mod, automatic installer, went for the full version (EUI included)

2 I activated all the mods provided in the full version (Community Patch, Community Balance Overhaul, City-State Diplomacy Mod, C4DF - CBP, More Luxuries).
Didn't add any other party mod just because I wanted to see what might cause these issues.
I have all the required DLCs (Civs, Ancient World Wonders, BNW).

3 Heavy graphic glitches, that is why I am leaning to think it might be linked to the EUI installation. As described, I moved the Community Balance Overhaul - Compatibility Files (EUI) folder from the MODS folder to the steamapps/common/Sid Meier's.../Assets/DLC folder. Please, take a look at the screenshots.

4 I used the automatic installer, everything wen smooth. beforehand, I also cleared the cache, the mods and the mod user data folders, trying to get a "clean" install.

---------------------------
Supporting information:
Please note that you can attach .zip files by dragging-and-dropping them. If possible, zip up all supporting data and post that way.

1. Log files (always attach your Logs folder, located at My Documents/My Games/Sid Meier's Civilization 5. Make sure you have enabled logging before experiencing an error! Go here to find out how: http://forums.civfanatics.com/showthread.php?t=487482):

2. Save game (always attach a save that was made a turn before the error; located at My Documents/My Games/Sid Meier's Civilization 5/ModdedSaves):

3. CvMiniDump.dmp file (attach if experiencing a game crash. Located at Program Files/Steam/steamapps/common/Sid Meier's Civilization V):

4. Screenshots (optional):
![glitch i](https://cloud.githubusercontent.com/assets/25144067/21971919/4dcb0924-dbf0-11e6-871a-08528cac82c2.jpg)
![glitch ii](https://cloud.githubusercontent.com/assets/25144067/21971920/4dd5af32-dbf0-11e6-96f4-c6c219f04ae8.jpg)

**Comment by kjades** [comment]
Quick update, tried installing Vox Populi without EUI this time, still get the same graphic issues.
Don't really know what to think know :|

**Comment by ErikSteiner** [comment]
Is GPU decompression activated in graphics settings? Try to turn it off.

**Comment by kjades** [comment]
Thanks for the reply. It's always been unticked.

**Comment by kjades** [comment]
Last update, reinstalled Vox Populi WITH EUI this time, activated only the Community Patch.
No graphic glitches, the mod seems to work perfectly.
Too bad I'd like to enjoy the full content (game mechanics changes, etc etc...)
Any ideas?

**Comment by bigcat88** [comment]
maybe try to reinstall GPU driver. and what DX u use? if u use DX 10 then try to switch to DX9.

**Comment by kjades** [comment]
I'll try to reinstall the GPU driver (sorry, half-noob here, you mean the graphic card drivers, don't you? :D)
Anyway, I'm using the DirectX 11 (middle option in the first window as the game starts), I'll definitely try the DX9 as you suggest.
Thanks.

**Comment by LoneGazebo** [comment]
@kjades We don't mess with the graphics engine, as we do not have access to it. My guess is low system memory or low GPU memory. Try turning your gfx settings down.

G

**Comment by kjades** [comment]
Thanks for the hint Gazebo, will surely give it a try and give you a feedback on this matter.

**Comment by kjades** [comment]
Latest update: the mod is finally working. I updated the graphic drivers, tuned down the gfx settings (the texture quality) AND I switched to DirectX 9. Not really sure which one of these did the trick, still...I can finally play this little wonder. :)
Thanks everybody for the help

**Comment by Vyyt02** [comment]
Me and a couple of other users have had a similar issue in the past. You can safely go back to DirectX 11 and then you can try gradually increasing various graphics settings every time you launch the game. If I remember correctly, it was the texture quality or Anti-Aliasing ... cannot remember or find the thread where we solved the issue, but the safe approach is increasing the quality until the glitches re-appear - careful, they might not appear immediately.

**Comment by ashendashin** [comment]
I keep my texture quality on high. I think it depends on your system, as I've never seen graphical issues on the setup screen. As Vyyt said, get back to DX11 and find out which settings you're comfortable with and doesn't seem to cause problems.

**Comment by Vyyt02** [comment]
You probably noticed it, but just to make sure, @ashendashin - this is probably a different kind of graphical glitches from the ones we have had recently - for some reason this one only affects icons. We had pin-pointed the exact settings which were causing it about a year ago, but I cannot find the damn thread - neither here, nor at CivFanatics - I only found the newer ones where I am suggesting the gradual increasing of various settings and then people saying "yay it worked!".

@kjades, if you find the exact settings, please do let us know - so that we can help better when someone runs into the same problem in the future again. It seems to be quite rare and not limited to weak hardware.

**Comment by kjades** [comment]
Thanks to both for the help,
I understand your point, and agree, even though the glitch also affected in-game graphics once the match started. I will doublecheck by starting a new match with DX11 and tuning the graphic settings to minimum (playing with DX9 causes no problem even with higher settings) and increasing it step by step to see where the issue actually lies.
I also agree that it might be due to the low level hardware of the cpu I'm playing on, even though I'm not completely sure.

**Comment by kjades** [comment]
Ok, so, latest news:
doublechecked by selecting DX11 and turning graphics to the absolute minimum.
Still got the glitch.
So, it's rather safe to say that (at least in my case) it's not a matter of graphic settings, just about DirectX.
Still, it's quite certain that the low-end piece of machinery I'm playing on might have a part in this. :)

**Comment by ErikSteiner** [comment]
In your initial post, you wrote that you moved the " Community Balance Overhaul - Compatibility Files (EUI)" folder to the DLC folder. This is wrong. Please navigate to:
common\Sid Meier's Civilization V\Assets\DLC

Make sure that only these folders are present and delete any folder that is not present at the screenshot below.
![folder](https://cloud.githubusercontent.com/assets/2684018/22143242/a214941c-def9-11e6-85f6-a30f621ff676.png)

1. Start Steam and right click Civilization V and hit Properties. Click "local files" and validate your installation.
2. Navigate to: Documents\My Games\Sid Meier's Civilization 5
Delete files: config.ini, GraphicsSettingsDX9.ini, GraphicsSettingsDX11.ini, UserSettings.ini
Delete folders: Cache, ModUserData and logs
3. Navigate to: My Games\Sid Meier's Civilization 5\Mods
If you are only using the modules of Vox Populi, you should have six folders there. Delete them. If you have more or less folders, delete them. Make sure that the Mods folder is empty afterwards.
4. Download and install Vox Populi Automatic Installer from the How to install thread. If you still have the installer on your hard drive use it instead. Select Vox Populi (including EUI) and hit Next until you can hit Install.
5. Then navigate to: My Games\Sid Meier's Civilization 5\Mods
Make sure that this folder is containing six folders now.
6. Then navigate to: common\Sid Meier's Civilization V\Assets\DLC
In addition to the folders of the screenshot above, you should see a folder called UI_bc1
7. Start Civilization V and go to Options, deactivate MSAA and GPU processing, set your resolution to the correct value. Hit Apply.
8. In the main menu go to Mods and wait a second or two until the mods are sorted. Activate every module and hit next. Create a game. I hope it is working now.

If it is not working, what Graphics Card do you have?

**Comment by kjades** [comment]
Hi,
sorry for the very late reply, just had some time to give a try to your previous hints.
So, I followed your instructions step by step to install Vox Populi on two different computers, just, you know, to doublecheck.
On the first one, my private laptop, the installation went smooth, and no issue at all appears, I can play VP with DirectX11 without any graphic glitch. I have to say though that the issue I had before was not related to this machine, as I had never installed VP on my laptop.

The issue as I have described it so far appeared on my working desktop pc, so once again, I reinstalled VP following your instructions.
The issue is still there. Selecting DirectX 11 causes the graphic glitch. To be sure, I also selected the very same graphic settings on both computers, even though the desktop is supposed to be more powerful than my laptop. Needless to say, reverting to DirectX 9 brings up no issue at all, no graphic glitch when playing. At this point I think it's rather safe to say it's a machine-related issue, something wrong with my hardware-software specs maybe. At least, that's my best guess.
